hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -125,7 +125,10 @@ int otbn_stack_element_peek(int index, svBitVecVal *val) __attribute__((weak));
 // described above).
 extern "C" unsigned otbn_model_step(OtbnModel *model, svLogic start_i,
                                     unsigned start_addr, unsigned status,
-                                    svBitVecVal *err_bits /* bit [31:0] */);
+                                    svBitVecVal *err_bits, /* bit [31:0] */
+                                    svLogic edn_rnd_data_valid_i,
+                                    svLogicVecVal *edn_rnd_data,
+                                    svLogic edn_urnd_data_valid_i);
```

Comment:
Unlike in SystemVerilog code, we haven't been using the `_i` suffix in C++. Would you mind dropping it? (Especially as `edn_rnd_data` seems to be different). Also, could you add a comment to show the width of `edn_rnd_data` like we do for `err_bits`?

Reply:
Sure, also removed it from `start_i` (as it having it was the reason I kept it in the first place)

Reply:
Oh! I hadn't noticed I'd left that in! Looks good to me.